spec/bundle/brew_dumper_spec.rb

```diff
@@ -14,6 +14,7 @@
                     name:                   "foo",
                     desc:                   "foobar",
                     oldname:                "oldfoo",
+                    oldnames:               ["oldfoo"],
```

Comment:
Shouldn't the previous line also be deleted in this case?

Reply:
It could, but the method is still present on the class this stub stubs.